### ISSUE TITLE
[TIP] Missing dependencies for 64bit platform

### ISSUE THREAD

**Issue by Draqun** [ValveSoftware/Proton#843]
Hi.
Maybe this was reported but I don't have time to search it. When you install drivers for 64bit platform there also should be installed libnvidia-gl-396:i386. Without this package there are problems with running Steam on Ubuntu\Mint. I mean there is error

> glxChooseVIsual failed

Best regards.
Draqun

**Comment by mamoit** [comment]
[Here](https://github.com/ValveSoftware/Proton/blob/proton_3.7/PREREQS.md) are the instructions for the Ubuntu drivers.
If you don't have time to research, please don't open a ticket until you do.
This issue list is getting quite crowded, and we should try to keep it as concise as possible.
I have kubuntu running also on amd64 and I don't need libnvidia-gl:i386, just amd64.

**Comment by kisak-valve** [comment]
It's impressive that you were able to run the steam client at all with broken 32 bit opengl in order to hit this issue with Proton.

Package dependencies have to be managed in Steam's package for your distro's package management because the specific names of the packages varies wildly between distros.

Closing as out of scope for Proton.
